### PR TITLE
add ChildCaseCyclePayments model, controller, api, swagger info

### PR DESCRIPTION
**Pull request by weedySeaDragon**
# 💅🏼 What issue does this fix?

#166 

## 🍂 Before You Submit

* [x] Did you write tests?
* [x] Did you run `bundle exec rspec` from the root?
* [x] Did you run `bundle exec rails rswag` from the root?
* [x] Did you run `bundle exec rubocop` from the root?
* [x] Did you run `yarn lint` in `/client`?
* [x] Did you run `yarn test` in `/client`?
* [x] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment Considerations

* [ ] Data Migrations
* [x] Schema Migrations
* [ ] Dependencies

## 🕯 Caveats, concerns

1. a number of models set their `slug` based on the `id`.  I created a protected method in ApplicationRecord that child classes can call: `set_slug_from_id`.  ChildCaseCyclePayment uses it: `before_save :set_slug_from_id`

2. For all of the factories, I replaced all user association with a `confirmed_user` so that the **confirmation email is not created.**  (It slows things down and fills up the test.log).
It's clear if you look at the factories.  I couldn't think of a reason not to use a confirmed_user.

### CODE REVIEW

File: app/models/application_record.rb [hunk 1]
```diff
@@ -13,4 +13,10 @@ def generate_slug(hash_string)
   rescue ActiveRecord::RecordNotUnique
     Digest::SHA1.hexdigest("#{hash_string}#{SecureRandom.uuid}")[8..14] if hash_string
   end
+
+  protected
+
+  def set_slug_from_id
```
Comment (katelovescode) [inline app/models/application_record.rb:19]: We're actually going to be getting rid of slugs in a later ticket, per our discussion in Slack - I don't think you need to change this now but we're gonna end up pulling it out. @csprayregen can we prioritize the slug ticket soonish?

File: spec/factories/businesses.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
   factory :business do
     name { Faker::Name.child_care_businesses }
     license_type { Licenses.types.keys.sample }
-    user
+    user factory: :confirmed_user
```
Comment (katelovescode) [inline spec/factories/businesses.rb:7]: 👍🏼😍👍🏼😍👍🏼😍

**Review by katelovescode — APPROVED** [review]
Looks great!

File: app/models/application_record.rb (hunk 1 above)
Comment (weedySeaDragon) [reply app/models/application_record.rb:19]: just saw that.  Yup -- we can decide which way to go (uuids v. slugs) and go from there.

**Pull request merged**